cypress-tests/cypress/support/commands.js

```diff
@@ -971,6 +993,7 @@ Cypress.Commands.add(
       if (response.status === 200) {
         expect(response.headers["content-type"]).to.include("application/json");
         globalState.set("paymentID", paymentIntentId);
+        globalState.set("connectorId", response.body.connector);
```

Comment:
Not needed.

Reply:
For routing there is no where we are get to know which connector the testcases are running. Do you remember we already discussed about this